products/ward round documentation/new feature description_ward round documentation.md

```diff
@@ -0,0 +1,248 @@
+    - Image documentation (e.g. for wound management)
+
+___
+
+## 4. User Stories
+
+    - List several user stories or scenarios, which are brief descriptions of how different user roles (doctors, nurses, pharmacists, administrative staff etc.) will interact with the feature.
+    - Each user story should follow the format: "As a [user role], I want to [action] so that [benefit]." (e.g., "As a doctor, I want to quickly access a patients' medication history so that I can make informed prescribing decisions.")
+
+- What types of ward rounds are there?
+    1. Doctors daily round on each patient in their ward. Ward Rounds usually include all junior doctors that are assigned to the ward as well as some senior doctors. It is common that nurses join the ward round only for those patients that they are responsible for, so that they can give additional information if needed.
+    2. Nurses usually round on patients at the beginning of their shift to pass on relevant information between from one shift to the next. The relevant information differs from the doctor ward rounds. Usually there are no decisions to be made regarding diagnostics and therapy.
+    3. Interdisciplinary ward rounds take place in settings such as ICU (intensive care unit) and IMC (intermediate care unit). This might include the following employee groups: doctors, nurses, physiotherapist, speech therapist, patient management team (admin), etc.
+
+**Main user story:**
+
+- > As a physician, I want to see a summary of all relevant information at a glance, with an emphasis on the latest developments, so that I do not miss important facts. I want to be able to present a complete picture of the patient's current situation so that the medical team can agree on the next steps to ensure that the patient receives the best possible medical care.
+
+___
+
+## 5. User Flow
+
+    - A graph that describes the flow of the user throughout the feature should be drawn. Where does the user access the feature? What can he or she do with it? What are possible actions within the process? Exact definitions of the views are not (yet) needed.
+
+
+![Ward Round Documentation: User Flow](images%20and%20documents/Ward%20Round%20Documentation_User%20Flow_v1.png)
+___
+
+## 6. Functional Requirements
+
+    - List all the specific functionalities the feature must have.
+    - Be as clear and detailed as possible, considering different use cases.
+    - Use common / user-friendly terms whenever possible, but include technical ones if necessary.
+
+- Ward Round Documentation interface:
+    - challenge: designing the ward round documentation interface in a way, that the relevant information can be seen without having to perform further actions but also not overcrowding the view
+    - Main view with prioritized information arranged in boxes (no scrolling!)
+    - Additional information (e.g. appointments, scores and scales, extensive lab results and not only abnormal values)
```

Comment:
again don't get too specific on the UX outlines, we have external teams for that. Focus on what you want to do during a round visit. Maybe shift some text from the upper part: Keep records about findings, access and review former findings, former reports etc. You want to be able to save the document, to edit it again, to delete it if necessary, to attend other documents, maybe files like pictures. Think about these functional requirements